packages/eas-cli/src/credentials/manager/ManageAndroid.ts

```diff
@@ -163,29 +161,21 @@ export class ManageAndroid {
     );
     const appLookupParams = await getAppLookupParamsFromContextAsync(ctx, gradleContext);
     if (action === AndroidActionType.CreateKeystore) {
-      const selectBuildCredentialsResult = await new SelectAndroidBuildCredentials(
+      const keystore = await new CreateKeystore(appLookupParams.account).runAsync(ctx);
+      const selectBuildCredentialsResult = await new CreateAndroidBuildCredentials().runAsync();
+      await ctx.android.createAndroidAppBuildCredentialsAsync(ctx.graphqlClient, appLookupParams, {
+        ...selectBuildCredentialsResult,
+        androidKeystoreId: keystore.id,
+      });
+    } else if (action === AndroidActionType.SetDefaultKeystore) {
+      const setDefaultBuildCredentialResult = await new SetDefaultAndroidKeystore(
```

Comment:
The name SetDefaultAndroidKeystore suggest that this action will update the default where in fact it only prompts whether it should. Either:
- move updating default keystore to the SetDefaultAndroidKeystore
- rename SetDefaultAndroidKeystore to sth more apropiates
(I suggest first option)